presentation/src/main/java/com/knocklock/presentation/MainActivity.kt

```diff
@@ -19,22 +24,73 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        if (!permissionGranted()) {
-            val intent = Intent(
-                "android.settings.ACTION_NOTIFICATION_LISTENER_SETTINGS"
-            )
-            startActivity(intent)
-        }
+        checkNotificationPermission()
+        requestPermission()
+        startService()
 
         setContent {
+            val settingViewModel: SettingViewModel = hiltViewModel()
             KnockLockTheme {
-                LockScreenRoute()
+                SettingRoute(
+                    viewModel = settingViewModel,
+                    onMenuSelected = { /*TODO*/ },
+                    onBackPressedIconSelected = { /*TODO*/ }
```

Comment:
```suggestion
                SettingRoute(
                    onMenuSelected = { /*TODO*/ },
                    onBackPressedIconSelected = { /*TODO*/ }
```
default 값으로 설정 되어있어서 빼도 좋을것 같아용

Reply:
@KimHance pull받아서 적용했습니당